test/gc/Transform/deepTileContractionNamedOp.mlir

```diff
@@ -0,0 +1,157 @@
+// RUN: gc-opt --split-input-file --deep-tile-contraction-named-op %s | FileCheck %s
+
+// -----
+
+/// CHECK-LABEL: @matmul_2Dx2D_f32
+func.func @matmul_2Dx2D_f32(%arg0: tensor<4096x4096xf32>, %arg1: tensor<4096x4096xf32>) -> tensor<4096x4096xf32> {
+    %cst_0 = arith.constant 0.000000e+00 : f32
+    %0 = tensor.empty() : tensor<4096x4096xf32>
+    %1 = linalg.fill ins(%cst_0 : f32) outs(%0 : tensor<4096x4096xf32>) -> tensor<4096x4096xf32>
+    // CHECK: scf.forall
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.forall
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.for
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.extract_slice
+    // CHECK: linalg.transpose
+    // CHECK: tensor.expand_shape
+    // CHECK: scf.if
+    // CHECK: linalg.fill
+    // CHECK: linalg.batch_reduce_matmul
+    // CHECK: else
+    // CHECK: linalg.batch_reduce_matmul
+    // CHECK: tensor.insert_slice
+    %2 = linalg.matmul {MThreads = 4 : i32, NThreads = 2 : i32,  KThreads = 1 : i32, MBlock = 256 : i32, NBlock = 256 : i32, KBlock = 256 : i32,innermostMBlock = 32 : i32, innermostNBlock = 32 : i32,  innermostKBlock = 32 : i32 } ins(%arg0, %arg1 : tensor<4096x4096xf32>, tensor<4096x4096xf32>) outs(%1 : tensor<4096x4096xf32>)  -> tensor<4096x4096xf32>
+    return %2 : tensor<4096x4096xf32>
+}
+
+// -----
+
+/// CHECK-LABEL: @matmul_4Dx4D_bf16
+func.func @matmul_4Dx4D_bf16(%arg0: tensor<128x128x32x32xbf16>, %arg1: tensor<128x128x16x32x2xbf16>) -> tensor<128x128x32x32xbf16> {
+    %cst_0 = arith.constant 0.000000e+00 : bf16
+    // CHECK: tensor.empty() : tensor<128x128x32x32xbf16>
+    %0 = tensor.empty() : tensor<128x128x32x32xbf16>
+    // CHECK-NOT: linalg.fill
+    %1 = linalg.fill ins(%cst_0 : bf16) outs(%0 : tensor<128x128x32x32xbf16>) -> tensor<128x128x32x32xbf16>
+    // CHECK: scf.forall
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.forall
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.empty() : tensor<8x8x32x32xf32>
+    // CHECK: scf.for
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.extract_slice
+    // CHECK: scf.for
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.extract_slice
+    // CHECK: tensor.extract_slice
```

Comment:
Is this FileCheck too relax to check missing information, saying `OFFSET` and `TILESIZE`?

Reply:
Thanks for the reminder! I add more check in the test